src/Markdown.js

```diff
@@ -1,11 +1,44 @@
 import React from "react";
 import ReactMarkdown from "react-markdown";
+import toc from "remark-toc";
 import CodeBlock from "./CodeBlock";
 
-const renderers = { code: CodeBlock };
+function flatten(text, child) {
+  return typeof child === "string"
+    ? text + child
+    : React.Children.toArray(child.props.children).reduce(flatten, text);
+}
+
+// Using logic from markdown-toc
+const slugify = text => {
```

Comment:
Please follow the Deno style guide ("Top level functions should not use arrow syntax
") and use `function slugify(text) {` here